internal/broker/broker.go

```diff
@@ -76,7 +76,7 @@ type session struct {
 	authModes         []string
 	attemptsPerMode   map[string]int
 
-	oidcProvider          *oidc.Provider
+	oidcServer            *oidc.Provider
```

Comment:
Ok, I see how you fixed the provider issue now...

However, is this name "accurate" enough? IIRC, the reason I used provider is because the term was homogeneous along all documentations (OpenID, OAuth, MS Entra, Google, etc.), so it wouldn't confuse external contributors/users.

Reply:
It's not perfect but better than what we had previously IMO. I think ideally, the `*oidc.Provider` should be part of the implementations of our own `Provider` interface, because they do belong together conceptually. That would require much more refactoring though.

I went with `oidcServer` because even the doc comment of the `oidc.Provider` struct says that it represents an OIDC server's configuration: https://github.com/coreos/go-oidc/blob/v3.11.0/oidc/oidc.go#L95-L95

Alternatively, we could name this field just `oidc` (and rename the `oidcCfg` field to `idTokenVerifierCfg`, because that's what it is according to the [doc comment](github.com/coreos/go-oidc/blob/v3.11.0/oidc/verify.go#L78-L78)).